OmniKitUI/ViewControllers/ReplacePodViewController.swift

```diff
@@ -26,27 +26,20 @@ class ReplacePodViewController: SetupTableViewController {
             case .normal:
                     break // Text set in interface builder
             case .fault(let faultCode):
-                if let faultType = faultCode.faultType {
-                    switch faultType {
-                    case .reservoirEmpty:              // $18
-                       instructionsLabel.text = LocalizedString("Empty reservoir. Insulin delivery has stopped. Please deactivate and remove pod.", comment: "Instructions when replacing pod due to an empty reservoir")
-                    case .exceededMaximumPodLife80Hrs: // $1C
-                       instructionsLabel.text = LocalizedString("Pod expired. Insulin delivery has stopped. Please deactivate and remove pod.", comment: "Instructions when replacing pod due to an expired pod")
-                    case .occluded,                    // $14
-                         .occlusionCheckValueTooHigh, .occlusionCheckStartup1, .occlusionCheckStartup2,         // $5A, $60, $60
-                         .occlusionCheckTimeouts1, .occlusionCheckTimeouts2, .occlusionCheckTimeouts3,          // $62, $66, $67
-                         .occlusionCheckPulseIssue, .occlusionCheckBolusProblem, .occlusionCheckAboveThreshold: // $68, $69, $6A
+                switch faultCode.rawValue {
+                case 0x18:
```

Comment:
We shouldn't use raw codes when we have a decoded enum version of them.  The logic for determining a localized description of a faultcode probably belongs on the FaultEventCode struct itself:

```
    public var localizedDescription: String {
        if let faultType = self.faultType {
            switch faultType {
            case .reservoirEmpty:
                return LocalizedString("Empty reservoir", comment: "Description for empty reservoir fault")
            case .exceededMaximumPodLife80Hrs:
                return LocalizedString("Pod expired", comment: "Description for pod expired fault")
            case .occluded,
                .occlusionCheckValueTooHigh, .occlusionCheckStartup1, .occlusionCheckStartup2,
                .occlusionCheckTimeouts1, .occlusionCheckTimeouts2, .occlusionCheckTimeouts3,
                .occlusionCheckPulseIssue, .occlusionCheckBolusProblem, .occlusionCheckAboveThreshold:
                return LocalizedString("Occlusion detected", comment: "Description for occlusion detected fault")
            default:
                return String(format: NSLocalizedString("The pod has detected internal fault. Code: %1$d", comment: "Format string for undescribed faults (1: The fault code value)"), rawValue)
            }
        } else {
            return String(format: LocalizedString("Unknown pod fault %1$d", comment: "Format string for unknown faults (1: The fault code value)"), rawValue)
        }
    }
```

Then, the code here becomes: 
```
instructionsLabel.text = String(format: LocalizedString("%1$@. Insulin delivery has stopped. Please deactivate and remove pod.", comment: "Format string providing instructions for replacing pod due to a fault"), faultCode.localizedDescription)
```

Reply:
+ I thought we wanted to display the actual pod fault # in decimal in the case of a fault that doesn't match anything and for matching fault #'s other than the Pod expired/Reservoir empty/Occlusion detected cases for which we have simplified messages which match the PDM (i.e., we don't want to display our internal fault code descriptions to the user in the replace pod screens, just in the Loop Report). So we really don't actually want to use any faultCode decoded strings here, do we?
+ The above example is based on the previous PR which used the .reservoirEmpty style case names which doesn't currently support printing the pod fault # for codes we don't have a mapping for. In the latest PR, FaultEventCode's rawValue is public as per the earlier recommendation so that all fault #'s can easily be reported as returned from the pod which is what I've always wanted to do and which you indicated you wanted as well. Do you want to abandon this approach and do something different now?
+ In any case, using the suggested code the build fails with "Value of type 'FaultEventCode' has no member 'localizedDescription'"

Reply:
Reworked to use a new faultCode.localizedDescription which returns a localized version of "Unknown pod fault %03d" (for a fault that Loop doesn't have an enum case for), "Occlusion detected", "Empty reservoir", "Pod expired",  or "Internal pod fault %03d" (for any other fault).